modules/configuring-knative-services-domains.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * serverless/getting-started-knative-services.adoc
+
+[id="configuring-knative-services-domains_{context}"]
+= Configuring domains for Knative services
+
+The Red Hat OpenShift Serverless Operator will attempt to find the public domain that is used by the cluster and use that same domain for Knative Services. Having a domain configured makes it possible to reach services in your cluster via HTTP over the internet directly, for example, using the `curl` command.
+
+Alternatively, you can configure a different domain for Knative services or even configure a different domain for different Knative services based on labels. Once you have completed this configuration, you can create a template for the domain configuration.
+
+.Procedure
+. To change the configuration for Knative services, you must edit the `config-domain` ConfigMap in the `knative-serving` namespace using `oc edit`.
+
+   # oc edit cm config-domain -n knative-serving
+
+. Copy the sample ConfigMap and modify the relevant fields.
+
```

Comment:
You need a `+` here to identify the source block as part of this command.

I'd also include `[source,yaml]` before this block.